pkgs/development/ocaml-modules/protoc/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, buildOcaml, ppx_deriving_protobuf }:
+
+buildOcaml rec {
```

Comment:
How did you find out about `buildOcaml`? Please use `mkDerivation`.

Reply:
I've seen that some packages use [buildOcaml](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/ocaml/default.nix), for example [textutils](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/ocaml-modules/textutils/default.nix)

Reply:
Don’t use `buildOcaml`, unless you want to package a legacy library from Jane Street at version 113.33 for OCaml 4.02.